src/fluree/server/config.clj

```diff
@@ -0,0 +1,195 @@
+(ns fluree.server.config
+  (:refer-clojure :exclude [load-file])
+  (:require [camel-snake-kebab.core :refer [->kebab-case-keyword]]
+            [clojure.java.io :as io]
+            [clojure.walk :as walk]
+            [fluree.db.util.json :as json]
+            [malli.core :as m]
+            [malli.transform :as transform]))
+
+(def registry
+  (merge
+   (m/predicate-schemas)
+   (m/class-schemas)
+   (m/comparator-schemas)
+   (m/type-schemas)
+   (m/sequence-schemas)
+   (m/base-schemas)
+   {::path string?
+    ::server-address string?
+    ::connection-storage-method [:enum
+                                 :ipfs :file :memory :s3 :remote]
+    ::indexing-options [:map
+                        [:reindex-min-bytes {:optional true} pos-int?]
+                        [:reindex-max-bytes {:optional true} pos-int?]
+                        [:max-old-indexes {:optional true} pos-int?]]
```

Comment:
`:max-old-indexes` can be zero, so I think you need `nat-int?` here instead of `pos-int?`